scraper.py

```diff
@@ -110,7 +110,7 @@ def scrape(self):
         urls = self.__collect_restaurants(10)
         for (name, url) in urls:
             self.driver.execute_script(f"window.open('{url}', '_blank');")
-            time.sleep(5)
-            # self.__get_summary()
             self.driver.close()
```

Comment:
On the first iteration, this will close the page with all the restaurants. Do we want to keep that tab open throughout?